src/general/OptionsMenu.cs

```diff
@@ -636,6 +653,40 @@ private void DisplayResolution()
             .FormatSafe(screenResolution.x, screenResolution.y);
     }
 
+    /// <summary>
+    /// Displays the GPU name, the display driver name and used video memory
+    /// </summary>
+    private void DisplayGpuInfo()
+    {
+        var videoAdapterName = VisualServer.GetVideoAdapterName();
+        gpuName.Text = TranslationServer.Translate("AUTO_GPU_NAME")
+            .FormatSafe(videoAdapterName);
+
+        if (OS.GetCurrentVideoDriver() == OS.VideoDriver.Gles2)
+        {
+            // Gles2 is being used
+            displayDriverName.Text = TranslationServer.Translate("AUTO_DISPLAY_DRIVER_NAME")
+                .FormatSafe(TranslationServer.Translate("GLES2"));
+        }
+        else
+        {
+            // Gles3 is being used
```

Comment:
It's not 100% certain, especially once we switch to Godot 4 so I think explicitly checking for GLES3 would be better and having an else case for "unknown" would be good.